### DISCUSSION TITLE
Feature Request - universal discipline-specific data entry forms

### DISCUSSION THREAD

**Discussion by mkoo** [ArctosDB/arctos#8052] [Ideas]
Issue created from #6784 
Mammals might be a good example for creating a common data entry template for specific collection types (eg.Mammals, Birds, Art, etc). 
Needs more thought and discussion but suggestion has merit and likely technically feasible

From @dustymc File an Issue. IDK how realistic 'MVZ Mammals' is (as "part of Arctos" - there's of course an API that'll support near anything), but 'mammals' might make a nice specialized form. (I think ~a dozen extra forms might be sustainable, I'm pretty sure 400 is not, but ?? - it seems an idea worth exploring if someone wants to.)

**Comment by ewommack** [comment]
> it seems an idea worth exploring if someone wants to.)

I know my students would have love to see some specialized forms for different taxons. What do you need to help with?

**Comment by dustymc** [comment]
> What do you need to help with?

I'm putting ~~two~~ three things first

1: Direction: Do we want to support multiple UIs or should we focus on one or ??? Multiple seems worth trying to me but I'm not the 'customer.'


Still 1: Scope. Remembering that there will always be a default 'everything' form, and that a specialized form should probably strive to deal with ~90% or such - "normal" - entry, whatever that is - what do we need? Eg do we need one all-collections bird form (maybe??), or can we go up to a "vertebrate" form (probably not), or do we need multiple bird forms (might be possible), or one for each bird collection (probably needs more resources), or one for each bird taxon (Step One: Hire like 50 more people....) or ???????????????

Also 1: Format. Are HTML forms (something like the current UI) the best way to address this, or should we be looking at something more spreadsheet-like, or ???????? Sky's the limit, but let's don't build things that ya'll don't want to use!

2: Content/functionality - I'll need a clear and open communication line with whoever's guiding this (which ideally I think is probably 'all the bird collections' or thereabouts, but I suppose I'll take what I can get!)

Or maybe that's all craycray, IDK, I'm up for whatever.

**Comment by dustymc** [comment]
Tabling for lack of interest/specifics.

**Comment by campmlc** [comment]
Reopening as there is certainly need and interest, just not enough bandwidth. This could be something that a reports developer or UI programmer could assist collections with.
@mkoo
